### PR TITLE
OPS-2407 syncronize instance-group template with remote state store format

### PR DESCRIPTION
**Pull request by cytopia**
## Upcoming git tag: `v1.2.1`

In order to dry-run the

1. state-store update
2. cluster update

we first need to bring the local cluster files (cluster.yml and instance-groups.yml) into the same format as they are stored in the state-store (on s3)

This was recently prevented by the following bug:

https://github.com/kubernetes/kops/issues/2881
However, it got fixed now

https://github.com/kubernetes/kops/pull/3053
So now we can combine all instance group templates in one.

---

The syncronization of the yaml format is useful for the planned diff (local config vs currently applied state store values):

## Preview
```bash
$ yamldiff.py <(kops get cluster -oyaml --state s3://common-cluster-internet-state-store 2>/dev/null) \
   build/common-kubernetes-internet.k8s.local/cluster.yml
```
```diff
--- /dev/fd/63
+++ build/common-kubernetes-internet.k8s.local/cluster.yml
@@ -4 +4 @@
-  creationTimestamp: 2018-06-22 14:38:56
+  creationTimestamp: null
```

### CODE REVIEW

File: templates/kops-create.sh.j2 [hunk 1]
```diff
@@ -61,22 +61,17 @@ echo
...
-	kops create --state ${KOPS_STATE_STORE} -f "${SCRIPTPATH}/ig-master.yml"
-fi
-
-echo "kops create --state ${KOPS_STATE_STORE} -f ig-worker.yml"
-if [ "${RUN}" -eq "1" ]; then
-	kops create --state ${KOPS_STATE_STORE} -f "${SCRIPTPATH}/ig-worker.yml"
+	kops create --state ${KOPS_STATE_STORE} -f "${SCRIPTPATH}/instance-groups.yml"
 fi
 
 echo "kops create secret --state ${KOPS_STATE_STORE} --name ${NAME} sshpublickey admin -i ./ssh-key.pub"
```
Comment (snovikov) [inline templates/kops-create.sh.j2]: I think you have to create secret before creating instance-groups (because of bastion host inside)

Comment (cytopia) [reply templates/kops-create.sh.j2]: This works fine. All those commands do are updating the configuration on S3. For that it should not matter in what order. The key is to have the following command at the very end, which in fact reads the configuration from S3 and then applies it:

```
kops update cluster --state ${KOPS_STATE_STORE} --name ${NAME} --yes
```

Comment (snovikov) [reply templates/kops-create.sh.j2]: Correct.

File: templates/instance-groups.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,97 @@
+{# ========================================================================= #}
```
[the remark this answers is not in the document]
Comment (snovikov) [reply templates/instance-groups.yml.j2]: It is not important, so let's skip it.

**Pull request merged**